coriolis/osmorphing/windows.py

```diff
@@ -221,3 +221,34 @@ def _create_service(self, key_name, service_name, image_path,
              "service_account": service_account,
              "start_mode": start_mode},
             ignore_stdout=True)
+
+    def run_user_script(self, user_script):
+        if len(user_script) == 0:
+            return
+
+        script_path = "$env:TMP\\coriolis_user_script.ps1"
+        try:
+            utils.write_winrm_file(
+                self._conn,
+                script_path,
+                user_script)
+        except Exception as err:
+            LOG.exception(err)
+            raise exception.CoriolisException(
```

Comment:
NIT: Considering we decided to only support Python 3, you can remove the LOG.error call and just say raise exception.CoriolisException(...) from err to preserve both stack traces together